src/VariableInput/test/storySettings.js

```diff
@@ -0,0 +1,7 @@
+import { Category } from '../../../stories/storiesHierarchy';
+
+export const storySettings = {
+  category: Category.COMPONENTS,
```

Comment:
Until we'll have the UX review, let's mark as WIP:
```suggestion
  category: Category.WIP,
```